### DISCUSSION TITLE
Shuts down while input shaping (Rooted)

### DISCUSSION THREAD

**Discussion by Sensai-Black** [Guilouz/Creality-Helper-Script-Wiki#539] [Issues]
I was wondering if theres a fix for {"code": "key93", "msg":"MCU 'rpi' shutdown: Rescheduled timer in the past. Second time i tried input shaping calibrations, the "Klipper reports: SHUTDOWN

{"code": "", "msg":"MCU 'mcu' shutdown: Move queue overflow", "values": []}" appeared. The Root was successful & I'm at a stand still

**Comment by mindkill666** [comment]
try to reboot the printer, it has very limited memory on the motherboard, so when powered off and on, the memory refreshes and you should have no problem doing an input shaper.
